doc/source/io.rst

```diff
@@ -136,7 +136,12 @@ usecols : array-like or callable, default ``None``
   that correspond to column names provided either by the user in `names` or
   inferred from the document header row(s). For example, a valid array-like
   `usecols` parameter would be ``[0, 1, 2]`` or ``['foo', 'bar', 'baz']``. 
-  Element order is ignored, so ``usecols=[0, 1]`` is the same as ``[1, 0]``.
```

Comment:
can you add a blank line before the additions.

Reply:
Sure thing.  Please refer to 68efc4bd3acb20acacd65f887385591db62b8394